### PR TITLE
Wingman: Use infix notation for operator applications

### PR DESCRIPTION
**Pull request by isovector**
#1671 makes it clear that using prefix notation for operators is a horrendous atrocity. This PR makes Wingman generate infix notation whenever applying the first two arguments of an operator.

### CODE REVIEW

**Comment by isovector** [comment]
User error. I forgot to update my featureset.

**Comment by jneira** [comment]
Lets see if removing ghcide tests speed up ci

File: plugins/hls-tactics-plugin/test/golden/AutoInfixApplyMany.hs.expected [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test :: (a -> b -> x -> c) -> a -> (a -> b) -> x -> c
+test (/:) a f x = (a /: f a) x
+
```
Comment (Ailrun) [inline plugins/hls-tactics-plugin/test/golden/AutoInfixApplyMany.hs.expected:3]: Just an idea: it's maybe good to add a nested operators case, like
```Haskell
test :: (a -> b -> c) -> (c -> d -> e) -> a -> (a -> b) -> d -> e
test (/:) (//:) a f x = (a /: f a) //: d
```

File: plugins/hls-tactics-plugin/src/Wingman/CodeGen.hs [hunk 2]
```diff
@@ -203,3 +206,12 @@ buildDataCon should_blacklist jdg dc tyapps = do
     & #syn_trace %~ rose (show dc) . pure
     & #syn_val   %~ mkCon dc tyapps
 
+
+------------------------------------------------------------------------------
+-- | Make a function application, correctly handling the infix case.
+mkApply :: OccName -> [HsExpr GhcPs] -> LHsExpr GhcPs
+mkApply occ (lhs : rhs : more)
+  | all (liftA2 (||) isSymbol isPunctuation) (occNameString occ)
```
Comment (wz1000) [inline plugins/hls-tactics-plugin/src/Wingman/CodeGen.hs:214]: Use `isSymOcc`

**Review by wz1000 — APPROVED** [review]

**Pull request merged**